src/main/kotlin/no/nav/familie/ef/mottak/repository/SoknadRepository.kt

```diff
@@ -11,7 +10,7 @@ interface SoknadRepository : JpaRepository<Soknad, String> {
 
     fun findFirstByTaskOpprettetIsFalse(): Soknad?
 
-    fun findFirstByVedleggIsNotNull(): Soknad?
+    fun findFirstBySaksnummerStartingWith(json: String): Soknad?
```

Comment:
Hvorfor `StartingWith`?

Reply:
For det var letteste måten å se om det er en json-string. Starter med {